src/groupeddataframe/splitapplycombine.jl

```diff
@@ -1017,7 +1162,9 @@ end
 function _combine_with_first(first::Union{NamedTuple, DataFrameRow, AbstractDataFrame},
                              f::Any, gd::GroupedDataFrame,
                              incols::Union{Nothing, AbstractVector, Tuple},
-                             firstmulticol::Val)
+                             firstmulticol::Val, idx_agg)
+    extrude_case = false
```

Comment:
Why "case"?

Reply:
And what would you suggest. We have to keep track of two cases:
1. Case when we extrude value from `Ref` or 0-dimensional array
2. Case when we do not extrude it

Later we have to remember it because `first` can be a vector and still this is a single-row case (because we extruded this vector from e.g. `Ref`).

I leave it open as maybe you have a better name for it.

Reply:
I think I'd simply have said "extrude".

Reply:
OK - changed.